src/libraries/System.Threading.ThreadPool/tests/ThreadPoolTests.cs

```diff
@@ -1158,5 +1162,13 @@ public void ThreadPoolMinMaxThreadsEventTest()
 
         public static bool IsThreadingAndRemoteExecutorSupported =>
             PlatformDetection.IsThreadingSupported && RemoteExecutor.IsSupported;
+
+        private static bool getUseWindowsThreadPool()
+        {
+            AppContext.TryGetSwitch("System.Threading.ThreadPool.UseWindowsThreadPool", out bool UseWindowsThreadPool);
```

Comment:
Nit: naming convention
```suggestion
            AppContext.TryGetSwitch("System.Threading.ThreadPool.UseWindowsThreadPool", out bool useWindowsThreadPool);
```